.gitattributes

```diff
@@ -4,3 +4,14 @@ tests/data/*-win.php eol=crlf
 
 # Don't show bundled libraries as source files in GitHub
 bundle/**/* linguist-vendored
+
+/.github export-ignore
+/tests export-ignore
+/features export-ignore
+/bin export-ignore
+/phpunit.xml.dist export-ignore
+/composer.json export-ignore
+/composer.lock export-ignore
```

Comment:
```suggestion
```
There is no `composer.lock` in this repository.

https://github.com/wp-cli/wp-cli/blob/03d30d4138d12b4bffd8b507b82e56e129e0523f/.gitignore#L13